src/modules/Elsa.Workflows.Core/Services/WorkflowStateExtractor.cs

```diff
@@ -128,7 +128,7 @@ private static void ApplyActivityExecutionContexts(WorkflowState state, Workflow
         workflowExecutionContext.ActivityExecutionContexts = activityExecutionContexts;
         return;
 
-        ActivityExecutionContext? CreateActivityExecutionContext(ActivityExecutionContextState activityExecutionContextState)
+        async Task<ActivityExecutionContext?> CreateActivityExecutionContext(ActivityExecutionContextState activityExecutionContextState)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.